### ISSUE TITLE
h4 closing tag in example header template doesn't match

### ISSUE THREAD

**Issue by SilVdB** [anvilco/spectaql#134]
A very small issue, but I noticed the closing tag of the "Example" header in the template doesn't match: 

```
...
<section>
                  <h4>Example**</h3>**
                  <h5>Query</h5>
...
```

**Comment by newhouse** [comment]
Thanks @SilVdB !!! I'll have a look ASAP...

**Comment by newhouse** [comment]
Fix is out in version `0.9.2`...thanks again @SilVdB !
